lib/ui/src/settings/release_notes_page.tsx

```diff
@@ -1,16 +1,22 @@
 import { useStorybookApi } from '@storybook/api';
-import React, { FunctionComponent, useEffect } from 'react';
+import React, { FunctionComponent, useEffect, Fragment } from 'react';
 
 import { ReleaseNotesScreen } from './release_notes';
 
-const ReleaseNotesPage: FunctionComponent<{ onClose: () => void }> = ({ onClose }) => {
+const ReleaseNotesPage: FunctionComponent<{}> = () => {
   const api = useStorybookApi();
 
   useEffect(() => {
     api.setDidViewReleaseNotes();
   }, []);
 
-  return <ReleaseNotesScreen onClose={onClose} version={api.releaseNotesVersion()} />;
+  const version = api.releaseNotesVersion();
+
+  return version ? (
+    <ReleaseNotesScreen version={version} />
+  ) : (
+    <Fragment>Sorry, we don't know what version to load</Fragment>
```

Comment:
I'm not sure this is the solution to this problem. I would guess we instead hide the release notes menu/tab/page altogether if we don't have the version.

Reply:
yeah I'll address this in higher up the component tree